test/busted_runner.lua

```diff
@@ -9,4 +9,12 @@ if platform and platform.sysname:lower():find'windows' then
   end
 end
 
+local deps_install_dir = os.getenv 'DEPS_INSTALL_DIR'
+package.path = deps_install_dir.."/share/lua/5.1/?.lua;"..deps_install_dir.."/share/lua/5.1/?/init.lua;"..package.path
+if platform and platform.sysname:lower():find'windows' then
```

Comment:
I'd break out this as `local suffix = (platform and platform.sysname:lower():find'windows') and '.dll' or '.so'` and avoid using an if for the main expression.